wordle-web/src/scripts/wordleGame.ts

```diff
@@ -1,13 +1,24 @@
 import { Word } from '@/scripts/word'
 import { WordsService } from './wordsService'
-import type { Letter } from './letter'
+import { LetterStatus, type Letter } from './letter'
+import { Player } from './userService'
+import Axios from 'axios'
 
 export enum WordleGameStatus {
   Active = 0,
   Won = 1,
   Lost = 2
 }
 
+export interface GameResult {
+  playerId: number
+  playerName?: string
+  attempts: number
+  duration: number
+  success: boolean
+  score?: number
+}
+
```

Comment:
This probably has enough information to go in its own class, but I'm unfamiliar with TS conventions on that so I might be wrong